test/cppsuite/test_harness/workload/thread_context.cxx

```diff
@@ -285,6 +285,45 @@ thread_context::insert(
     return (true);
 }
 
+bool
+thread_context::remove(
+  scoped_cursor &cursor, uint64_t collection_id, const std::string &key, wt_timestamp_t ts)
+{
+    WT_DECL_RET;
+    testutil_assert(tracking != nullptr);
+    testutil_assert(cursor.get() != nullptr);
+
+    /*
+     * When no timestamp is specified, get one to apply to the update. We still do this even if the
```

Comment:
The word `update` might be ok as we are updating the collection in some way, but should we use another word to avoid confusion?